runtime/setup.py

```diff
@@ -412,35 +491,68 @@ def generate_version_py():
     ],
     url="https://github.com/openxla/iree",
     python_requires=">=3.8",
-    ext_modules=[
-        CMakeExtension("iree._runtime"),
-    ],
+    ext_modules=(
+        [
+            CMakeExtension("iree._runtime_libs._runtime"),
+        ]
+        + (
+            [CMakeExtension("iree._runtime_libs_tracy._runtime")]
+            if ENABLE_TRACY
+            else []
+        )
+    ),
     cmdclass={
         "build": CustomBuild,
         "built_ext": NoopBuildExtension,
         "build_py": CMakeBuildPy,
     },
     zip_safe=False,
-    package_dir={
-        # Note: Must be relative path, so we line this up with the absolute
-        # path built above. Note that this must exist prior to the call.
-        "": f"{CMAKE_INSTALL_DIR_REL}/python_packages/iree_runtime",
-    },
+    package_dir=combine_dicts(
+        {
+            # Note: Must be relative path, so we line this up with the absolute
+            # path built above. Note that this must exist prior to the call.
+            # "iree.runtime": "bindings/python/iree/runtime",
+            "iree.runtime": "bindings/python/iree/runtime",
```

Comment:
remove duplicate (commented out) line
```suggestion
            "iree.runtime": "bindings/python/iree/runtime",
```